analysis_tasks/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM r-base:3.2.5
+
+RUN R --slave --silent --vanilla \
+    -e 'install.packages(c("optparse", "caret", "plyr", "gbm", "doParallel"), repos="http://cran.rstudio.com")'
+
+RUN mkdir -p /opt/analysis_tasks
+WORKDIR /opt/analysis_tasks
+
+COPY . /opt/analysis_tasks
+
+# Provide access to the R environment by default if run without parameters
+CMD ["R", "--vanilla"]
```

Comment:
Changing this to use `ENTRYPOINT` and `CMD` might make simpler to run the container later. Say if it was always meant to be launched with `Rscript`, then all you'd need to supply to run would be the name of a file.